app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error message 'All friends should be vaccinated' is not very descriptive. According to the checklist, it's better to use descriptive error messages. You could provide more information, for example 'One of the friends is not vaccinated. All friends should be vaccinated to visit the cafe.'